Dockerfile-debian

```diff
@@ -0,0 +1,89 @@
+FROM debian:stretch-slim
+
+ARG tarball=amazon-corretto-8.222.10.1-linux-x64.tar.gz
```

Comment:
@duncaan Corretto supports .deb installer (see [Corretto 8 release page](https://github.com/corretto/corretto-8/releases)). Would you modify the dockerfile to download and install .deb file instead (See [Corretto 8 installation instruction for Debian-based Linux](https://docs.aws.amazon.com/corretto/latest/corretto-8-ug/generic-linux-install.html))? (Or let me know if you prefer .tar.gz for a reason).

Reply:
I updated using the install instructions there.